microdnf.spec

```diff
@@ -7,7 +7,7 @@ Summary:        Lightweight implementation of DNF in C
 
 License:        GPLv2+
 URL:            https://github.com/rpm-software-management/microdnf
-Source0:        %{url}/archive/v%{version}/%{name}-%{version}.tar.gz
+Source0:        %{url}/archive/%{version}.tar.gz?/%{name}-%{version}.tar.gz
```

Comment:
Take `.tar.gz?` out. It's redundant and unneeded.

Reply:
What do you mean?

Reply:
You mean that Github understands `%{url}/archive/%{version}.tar.gz?/%{name}-%{version}.tar.gz` and `%{url}/archive/%{version}/%{name}-%{version}.tar.gz` in the same way? Maybe you better fix it yourself as you want?

Reply:
Yes, it understands them the same way. It sets via content-disposition header what the filename should be.